features/fixtures/shared/scenarios/Scenario.m

```diff
@@ -45,10 +45,36 @@ - (instancetype)initWithConfig:(BugsnagConfiguration *)config {
     return self;
 }
 
+- (void)waitForNetworkConnectivity {
+    NSDictionary *proxySettings = (__bridge_transfer NSDictionary *)CFNetworkCopySystemProxySettings();
+    NSLog(@"*** Proxy settings = %@", proxySettings);
+    
+    // BrowserStack appears to use a proxy to establish connectivity with Maze Runner...
```

Comment:
I'm not sure we need to mention BrowserStack at all here.  You're using just the same protocol and URL as the notifier does, so it seems like a separate issue to me (in fact I don't think it's an issue at all, just a feature of the test environment).  If/when we move device farm the comment will be redundant anyway.

Reply:
Updated the comment